circle.yml

```diff
@@ -0,0 +1,16 @@
+dependencies:
+  cache_directories:
+    - /usr/local/Homebrew
+  override:
+    - brew install shellcheck
+
+machine:
```

Comment:
@iwz I don't have experience yet with Circle 2.0 so was shying away from it for now. Thoughts on what the benefits of it are?